index.html

```diff
@@ -6,13 +6,29 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="stylesheet" href="main.css" />
+    <script src="main.js"></script>
     <title>Natnael's portfolio Home page</title>
   </head>
   <body>
     <header>
       <nav>
-        <a href="#">Welcome</a>
-        <img src="images/more.png" alt="menu" />
+        <div class="mob-nav-head">
+          <a href="#" id="mnha">Welcome</a>
+          <img src="images/more.png" alt="menu" id="menu" onclick="display()" />
+        </div>
+        <div class="mob-nav-container" id="mnc">
+          <ul class="mob-nav">
+            <li>
+              <a href="#personal-works" onclick="menuClicked()">Portfolio</a>
+            </li>
+            <li>
+              <a href="#about-me" onclick="menuClicked()">About</a>
+            </li>
+            <li>
+              <a href="#contact-me" onclick="menuClicked()">Contact</a>
+            </li>
+          </ul>
+        </div>
```

Comment:
- [ ] You did a good job on the mobile menu functionalities but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design as much as possible. Please increase the size of the close(X) button and decrease the gap between mobile menu options and make sure it matches with the Template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/167462385-31b7f2ba-65a2-4678-aa42-58cce73938f0.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/167462423-aaf34fc2-4103-4c5d-a3b6-bd3c212bb711.png)